scripts/vesting.ts

```diff
@@ -0,0 +1,83 @@
+import BigNumber from "bignumber.js"
+import * as data from './data/rewards.json'
+
+type StaticVestingInfo = {
+    origin: string
+}
+
+type DynamicVestingInfo = {
+    destination: string,
+    schedule: {
+        amtToBeVestedOverTime: string,
+        start: string,
+        period: string,
+        per_period: string,
+        period_count: string 
+    }
+}
+
+type VestingBatch = {
+    statics: StaticVestingInfo
+    vestings: DynamicVestingInfo[]
+}
+
+const origin = '!!! address that pays bsx as they vest; treasury address I guess !!!' 
+// todo;  this is the address that pays out bsx as they vest, to target
+
+const thirtyPercentStart = '!!! end block of BSX LBP !!!'  
+// end block of BSX LBP, when is that ?
+
+const seventyPercentStart = '9071999'  
+// todo fetch; start block of lease ^^^ instead of hard-code
```

Comment:
should we start the vesting schedule at the exact start time of slot 15,  or should we start the schedule at the block where we won the parachain slot,  or the block where basilisk started producing blocks ?

Reply:
very valid question, cc @jak-pan but i think the answer is at the exact start time of slot 15.

Reply:
when we won parachain slot

Reply:
implemented with comment